### ISSUE TITLE
"Automatically fill-in single credentials entry" function should not fill in field already have value

### ISSUE THREAD

**Issue by kraml** [smorks/keepasshttp-connector#83]
I have this "Automatically fill-in single credentials entry" option turned on in settings. It works quite conveniently most of the time, but causes problem occasionally.

For example, when I'm on the home wireless router setup page, the keepasshttp-connector auto files my router login info for the wireless setup page, username filled in the SSID name field, password filled in the wireless password field. The first time this happened I failed to notice it and submitted the form, ended up changing my WIFI SSID and password :(

So I believe that as a general rule when a user/pass field already has content loaded, the keepasshttp-connector should not try to automatically fill/overwrite the content.

**Comment by smorks** [comment]
yes i have seen this behavior before too. i would say it's generally not a good idea to have the "Auto-fill" option on, but like you said, it would probably also be a good idea not to overwrite content too, if that option is on.
